pkg/sfu/downtrack.go

```diff
@@ -36,6 +36,7 @@ type TrackSender interface {
 	ID() string
 	SubscriberID() livekit.ParticipantID
 	TrackInfoAvailable()
+	HandleRTCPSenderReportData(layer int32, srData *buffer.RTCPSenderReportData)
```

Comment:
A new method in `TrackSender`. `sfu.DownTrack` does not do anything and has a shell implementation. But, this will send this data across in relay case. Will need an addition to the relay protocol also.